examples/iam-read-only-policy/versions.tf

```diff
@@ -0,0 +1,7 @@
+terraform {
+  required_version = ">= 0.15.0"
```

Comment:
Please set `>= 0.12.6` as on all other modules. We will bump to `>= 0.13.1` in all modules in another PR.

Reply:
done